src/Files.App/ViewModels/Properties/HashesViewModel.cs

```diff
@@ -23,11 +26,47 @@ public HashInfoItem SelectedItem
 		public Dictionary<string, bool> ShowHashes { get; private set; }
 
 		public ICommand ToggleIsEnabledCommand { get; private set; }
+		public ICommand CompareFileCommand { get; private set; }
 
 		private ListedItem _item;
 
 		private CancellationTokenSource _cancellationTokenSource;
 
+		private string _hashInput;
+		public string HashInput
+		{
+			get => _hashInput;
+			set
+			{
+				SetProperty(ref _hashInput, value);
+
+				OnHashInputTextChanged();
+				OnPropertyChanged(nameof(IsInfoBarOpen));
+				IsInfoBarOpen = !string.IsNullOrEmpty(value);
+			}
+		}
+
+		private InfoBarSeverity _infoBarSeverity;
+		public InfoBarSeverity InfoBarSeverity
+		{
+			get => _infoBarSeverity;
+			set => SetProperty(ref _infoBarSeverity, value);
+		}
+
+		private string _infoBarTitle;
+		public string InfoBarTitle
+		{
+			get => _infoBarTitle;
+			set => SetProperty(ref _infoBarTitle, value);
+		}
+
+		private bool _isInfoBarOpen;
+		public bool IsInfoBarOpen
+		{
+			get => _isInfoBarOpen;
+			set => SetProperty(ref _isInfoBarOpen, value);
+		}
```

Comment:
```suggestion
		public bool IsInfoBarOpen
		 => !string.IsNullOrEmpty(HashInput);
```
This change should eliminate the need for manual property assignment. To update this property, we can use `OnPropertyChanged` in the setter for `HashInput`.